### DISCUSSION TITLE
How can I see all available badges in GitHub?

### DISCUSSION THREAD

**Discussion by rodgeraraujo** [community/community#18620] [Other Feature Feedback, Questions, & Ideas]
Please Help!!

**Comment by Morph33** [comment]
badges are easter eggs

**Reply by Morph33** [reply]
https://github.com/orgs/github-community/discussions/18294

**Reply by matt6frey** [reply]
Gotta say I think these badges are a fun addition. 👍

**Reply by rodgeraraujo** [reply]
yeah, I think the idea of them being easter eggs is really fun

**Comment by primanegara** [comment]
Here is the more information about badges you can earn
https://github.com/Schweinepriester/github-profile-achievements

The badge you have can see on your overview page

**Comment by lethetai2605** [comment]
https://github.com/Schweinepriester/github-profile-achievements

**Comment by herkura** [comment]
After answering two discussions, you will get the "Galaxy Brain" badge.
Get Two PR's merged, and get the "Pull Shark" badge.
For the "YOLO" badge, get your PR reviewed by a collaborator in your repo.
Hope this helps 😄

**Reply by rodgeraraujo** [reply]
this is nice!

**Comment by krupalimakadiya** [comment]
Get Your Two PR's merged, and get the "Pull Shark" badge.

**Comment by SleazeStiKs** [comment]
### The Pull Shark and Galaxy Brain have different tiers. 

**Pull Shark**
- Tier 1 after 2 PRs merged
- Tier 2 for 16 PRs merged
- Tier 3 for 128 PRs merged
- Tier 4 for 1024 PRs merged

**Galaxy Brain**
- Tier 1 after 2 discussion answers
- Tier 2 for 8 discussion answers
- Tier 3 for 16 discussion answers
- Tier 4 for 32 discussion answers
